packages/giselle-engine/package.json

```
@@ -55,12 +55,15 @@
 		"@ai-sdk/google": "catalog:",
 		"@ai-sdk/openai": "catalog:",
 		"@ai-sdk/perplexity": "catalog:",
+		"@fal-ai/client": "^1.3.0",
 		"@giselle-sdk/data-mod": "workspace:^",
 		"@giselle-sdk/github-client": "workspace:^",
 		"@giselle-sdk/language-model": "workspace:^",
 		"@giselle-sdk/text-editor": "workspace:^",
 		"@giselle-sdk/usage-limits": "workspace:^",
 		"ai": "catalog:",
+		"langfuse": "3.37.0",
+		"langfuse-core": "^3.37.0",
```

"In this project, we're using pnpm Catalogs to manage the versions of dependent packages.

https://pnpm.io/catalogs

Please add the following to `pnpm-workspace.yaml`:

```yaml
langfuse: 3.37.0,
langfuse-core: 3.37.0
```

And then update this `package.json` accordingly."
```suggestion
		"langfuse": "catalog:",
		"langfuse-core": "catalog:",
```